mlops/parallelm/mlops/mlops_rest_connected.py

```diff
@@ -137,41 +139,34 @@ def get_last_approved_model(self, workflow_run_id, pipeline_inst_id):
                         ionId=workflow_run_id, pipelineInstanceId=pipeline_inst_id, modelType="lastApproved")
         return self._get_url_request_response_as_json(url)
 
-    def post_model_as_file(self, model_file_path, params, metadata):
+    def post_model_as_file(self, model):
         """
         Posts a file to the server
-        :param model_file_path: model file to upload
-        :param params: parameters dictionary
-        :param metadata: extended metadata(currently not used with rest connected)
+        :param model: model to publish
```

Comment:
Is model a path or a model object.. it is not clear from the doc

Reply:
ok
model object